### ISSUE TITLE
Team update - Jun 14, 2023

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#217]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by steff456** [comment]
**Thanks I'd like to give 🙌**
- Thanks @trallard and @gabalafou for the review in the PRs

**Updates from last week :heavy_check_mark:**
- Made review changes https://github.com/jupyter/accessibility/pull/126
- Made review changes https://github.com/jupyter/accessibility/pull/127
- Created https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/50
- Read the information about “rethinking the notebook cells”
- Worked in github dark - I hope by the end of today to have a draft PR up

**Challenges I faced and things I'd like assistance with 🙏**
- I'm still a little bit sick and my voice sounds weird

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- Is there more feedback to the documentation PRs? 
- Are we having an intern this year?

**Comment by gabalafou** [comment]
**Things I learned this week**
- You can use GitHub Codespaces to open repos in JupyterLab. This allows you to easily spin up JupyterLab to modify a notebook and then commit those changes directly from the JupyterLab UI because Codespaces includes the [jupyterlab-git](https://github.com/jupyterlab/jupyterlab-git) extension.

**Updates from last week :heavy_check_mark:**
- Finally submitted my review of an external contributor a11y PR to JupyterLab https://github.com/jupyterlab/jupyterlab/pull/14400#pullrequestreview-1455593106. This took a long time because I haven't done extensive screen reader testing and had to jump through some hoops on my Windows machine. But the win is that I'm in a better place to do this kind of review in the future. Thank goodness for GitPod.
- Took a close look at Tony's PR [the cell is landmark for a form #63](https://github.com/Iota-School/notebooks-for-all/pull/63)
- Met with Darian and Tony on Tuesday (recurring Rethinking Notebook Cells meeting) to think about how we can modify the notebook widget to output more accessible HTML for notebook in JupyterLab (and Notebook 7)
- Jumped briefly in triage call to discuss https://github.com/jupyter/notebook/issues/6800. I volunteered to break these into issues.

**My availability for next week**
- I'll be off on Monday, June 19

**Comment by trallard** [comment]
@gabalafou 
	- discussed review on the alt-text PR (using a screen reader and a PR at the same time)
	- Will be working on splitting `manfromjupyter`'s audit into individual issues, some caveats, and observations from the team
	  - there are a lot of helpful stuff and issues identified, though we do not know how this audit was done 
	  - perhaps we should re-title that issue to sound less prescriptive/absolute in terms of meeting WCAG conformance 

@steff456 
	- the github light PR should be able to be merged
	- next need to make an extension release and add the dark GH theme too  	
	- for the notebook authoring guide recommend reaching out to @melissawm to get input from someone authoring notebooks for the NumPy project

**Comment by trallard** [comment]
Notes about the authoring guides

- too long and difficult to follow (Isabela suggested grouping in terms of perceivable...)
- @gabalafou feels this is pretty much following WCAG docs/guides i.e., there is a lot of information, but it does not feel very approachable or focused on end-users (also need to go back to where this needs to live)
- the authoring guide can be contextualised (maybe by putting this into context within notebook examples)
